CHANGELOG.MD

```diff
@@ -0,0 +1,11 @@
+#Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+##[Unreleased]
+###Added
+###Changed
+##[x.x.x] - 2021-09-14 
```

Comment:
let's add a version for 2.0.9 with the proper changelog
https://github.com/microsoftgraph/msgraph-sdk-java-core/milestone/20?closed=1

Reply:
I was planning on doing this step when right before we merge this branch over to master since that's when the release would be triggered.

Reply:
That way I could also view all the commits/changes and make accurate additions to the changelog file.

Reply:
sure, that works for me as long as we don't forget about it :)